npm/ng-packs/packages/theme-shared/src/lib/components/form-input/form-input.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="mb-3">
+  <label class="form-label" *ngIf="label" [ngClass]="labelClass" [for]="inputId" > {{label | abpLocalization}} </label>
+  <input
+    class="form-control"  
+    type="text"
+    [id]="inputId" 
+    [placeholder]="inputPlaceholder" 
+    [readonly]="inputReadonly"
+    [ngClass]="inputClass" 
+    [ngStyle]="inputStyle" 
+    (blur)="onBlur.next()"
+    (focus)="onFocus.next()" 
+    [(ngModel)]="value">
```

Comment:
I thinki change value throw sepereately. othervise user can not attach the valueChanges (change) that

Reply:
AbstractNgModelComponent handles with valueChange